### DISCUSSION TITLE
how to use mcamara/laravel-localization in prefix

### DISCUSSION THREAD

**Discussion by battulga0719** [Z3d0X/filament-fabricator#165] [Q&A]
hello. i have a multiple locale and switch between and use different content depending on a current locale and i'm trying to use fabricator like that as well so can freely change contents. is there a solution for this currently it is returning an error saying A facade root has not been set.

use Mcamara\LaravelLocalization\Facades\LaravelLocalization;

return [
    'routing' => [
        'enabled' => true,
        'prefix' => LaravelLocalization::setLocale(),
    ],
];

**Comment by Voltra** [comment]
Would something like #119 help?
